client/src/components/Profiles.js

```diff
@@ -1,43 +1,91 @@
 import React, { useState, useEffect } from "react";
-import { Container, Message, Search, Table, Button } from "semantic-ui-react";
+import {
+  Message,
+  Search,
+  Table,
+  Button,
+  Grid,
+  Dropdown,
+  Input,
+  Container
+} from "semantic-ui-react";
 import { Link } from "react-router-dom";
-import { getProfiles } from "../api/profiles";
+import { getProfiles, getNationalities } from "../api/profiles";
 
 const Profiles = () => {
   const [profiles, setProfiles] = useState([]);
+  const [nationality, setNationality] = useState([]);
   const [searchInput, setSearchInput] = useState("");
+  const [selectDropdown, setSelectDropdown] = useState([]);
 
   const handleSearchChange = event => {
     setSearchInput(event.target.value);
   };
+  const handleOnChange = (e, data) => {
+    setSelectDropdown(data.value);
+  };
 
   useEffect(() => {
     getProfiles().then(response => {
       setProfiles(response);
     });
+    getNationalities().then(response => {
+      setNationality(response);
+    });
   }, []);
 
+  const options = nationality.map(national => ({
+    key: national.id,
+    text: `${national.id} -${national.nationality} `,
```

Comment:
I would just use "nationality" as the text. This shows "1 -Afghan" in the dropdown, for example.